src/views/Home/components/Milestones/milestones.scss

```diff
@@ -1,3 +1,85 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
 
-/* Delete these lines when you begin development */
+.Milestones {
+  margin: 0;
+  background: url("~images/components/milestones/background.svg") repeat-y, var(--dark-purple);
+  background-size: 100% auto;
+
+  &__container {
+    position: relative;
+    max-width: 1800px;
+    min-height: 400px;
```

Comment:
Would it be possible to change `max-width` to at least `2500px`? On an wide display, the pencil overlaps with the milestone boxes. We might also want to increase `min-height` to include all images at this screen resolution too.

Reply:
How does this look? I changed `max-width` to `2000px` instead because I feel like `2500px` is too wide. I also added a few more breakpoints for the images but that might still need tweaking. Please let me know if `flex-shrink` works!

<img src="https://user-images.githubusercontent.com/20177171/101300513-b5fc0700-37ea-11eb-9a76-4b736e18f79e.png" width="500px" />

Reply:
For some reason, it still looks weird on my end, but I'll trust your intuition! 
![image](https://user-images.githubusercontent.com/33771623/101439569-9daee980-38c9-11eb-9739-d3a4c88263fd.png)

Other than that, this looks good to me! Let's wait for @ilopezro or @mocklee's call!